src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,197 @@
+import { useState } from "react";
+import ham from "./ham.png";
+import logotext from "./logotext.png";
+
+export const NavBar = ({ selectTab }) => {
+  const handleToggle = () => setNavbarOpen((prev) => !prev);
```

Comment:
generally i have seen, all states are declared at start of function and then comes our local functions. I have seen that pattern in your code as well. You can declare useState first and then your handleToggle as it will be consistent with other files. Just a suggestion!